test/functional/web/safari-nativewebtap-e2e-specs.js

```diff
@@ -72,26 +76,28 @@ describe('Safari - coordinate conversion -', function () {
         this.timeout(MOCHA_TIMEOUT * 2);
 
         let driver;
+        const localCaps = _.defaults({
+          deviceName,
+          fullReset: true,
+          noReset: false,
+        }, caps);
         let skipped = false;
 
         before(async function () {
+          await B.delay(10000);
```

Comment:
why so?

Reply:
Oops! Because I wanted time to watch what was going on.